config/environments/production.rb

```diff
@@ -31,10 +30,24 @@
   # config.action_dispatch.x_sendfile_header = "X-Sendfile" # for Apache
   # config.action_dispatch.x_sendfile_header = "X-Accel-Redirect" # for NGINX
 
+  # Assume all access to the app is happening through a SSL-terminating reverse proxy.
+  # Can be used together with config.force_ssl for Strict-Transport-Security and secure cookies.
+  # config.assume_ssl = true
+
+  # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
+  config.force_ssl = true
```

Comment:
You'll want to check this one with @willfish and @amberstarlight  - they did some custom fix around cookies because they couldn't enable it previously I think. If it is enabled, I think the `.assume_ssl` option also needs enabling together with dropping the custom fix around cookie assignment